src/suites/cts/examples.spec.ts

```diff
@@ -0,0 +1,65 @@
+export const description = `
+Examples of writing CTS tests with various features.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+// To run these tests in the standalone runner, run `grunt build` or `grunt pre` then open:
+// - http://localhost:8080/?runnow=1&q=cts:examples:
+// To run in WPT, copy/symlink the out-wpt/ directory as the webgpu/ directory in WPT, then open:
+// - http://web-platform.test:8000/webgpu/cts.html?q=cts:examples:
+//
+// Tests here can be run individually or in groups:
+// - http://localhost:8080/?runnow=1&q=cts:examples:basic/async:
+// - http://localhost:8080/?runnow=1&q=cts:examples:basic/
+// - http://localhost:8080/?runnow=1&q=cts:examples:
+
+export const g = new TestGroup(GPUTest);
+
+g.test('basic', t => {
+  t.expect(true);
+  t.expect(true, 'true should be true');
+
+  t.shouldThrow(
+    () => {
```

Comment:
unit: Will the formatter let this function be one line?

Reply:
Unfortunately not, it's very opinionated.